### PR TITLE
Fix login deploy

### PR DESCRIPTION
**Pull request by BenjaminParment**
-> Moves the autologin function call to the Home component on mount
-> Connects header to redux to grab the login status so we dont need to pass down isLogginPage props anymore (just display logout function if loggedStatus == loggedIn)
-> Changes submissions endpoint to the right deployed one
-> Redirects user to loggedIn page if on the home component and logged in
-> RedirectURL for cognito is now just "url/" instead of "url/index.html" as the router redirects us automatically
-> Adds "Accept": "application/hal+json" to axios header to match the backend structure

### CODE REVIEW

File: src/components/header/Header.tsx [hunk 1]
```diff
@@ -23,10 +25,10 @@ class Header extends React.Component<ConnectedHeaderProps> {
                 <WebsiteLogo width={250} height={80} />
                 <hr className="vertical-line" />
                 <div className="title-website">Payments platform</div>
-                {this.props.isLoginPage && <span className="alpha-sign">ALPHA</span>}
+                {this.props.loggedInStatus === LoginStatus.loggedOut && <span className="alpha-sign">ALPHA</span>}
```
Comment (j-n-white) [inline src/components/header/Header.tsx]: So the alpha sign will disappear as soon as you press the login button?
```suggestion
                {this.props.loggedInStatus !== LoginStatus.loggedIn && <span className="alpha-sign">ALPHA</span>}

```

File: src/components/home/Home.tsx [hunk 2]
```diff
@@ -1,28 +1,53 @@
...
+        handleAutoLogin(
+            this.props.location.pathname,
+            this.props.setLoggingIn,
+            this.props.setLoggedIn,
+            this.props.setLoggedOut,
+        );
+    };
+
+    componentDidUpdate = (oldProps: ConnectedHomeProps) => {
+        if (oldProps.logginStatus !== LoginStatus.loggedIn && this.props.logginStatus === LoginStatus.loggedIn) {
```
Comment (j-n-white) [inline src/components/home/Home.tsx]: I think sticking to the more common pattern of checking if it has changed would make this easier to work with in the future.
```suggestion
        if (oldProps.logginStatus !== this.props.logginStatus && this.props.logginStatus === LoginStatus.loggedIn) {

```

File: src/components/home/Home.tsx [hunk 3]
```diff
@@ -1,28 +1,53 @@
...
 import Login from "components/login/Login";
 import { LoginStatus } from "enums/loginStatus";
 import React from "react";
-import { RootState } from "state/rootReducer";
 import { SANDBOX_PATH } from "sandbox/Sandbox";
 import { connect } from "react-redux";
+import { handleAutoLogin } from "components/login/login-service/loginService";
 
 const mapStateToProps = (state: RootState) => ({
     logginStatus: state.loginState.status,
```
Comment (j-n-white) [inline src/components/home/Home.tsx]: I'm not sure how I missed this before
```suggestion
    loginStatus: state.loginState.status,

```

File: src/components/header/Header.tsx (hunk 1 above)
Comment (BenjaminParment) [reply src/components/header/Header.tsx]: yeah, Catherine said so, well she actually said it should only appear on the login page so your suggestion is better

**Pull request merged**